ecommerce/frontend/src/utils.js

```diff
@@ -0,0 +1,71 @@
+export const productSortBy = {
+    'best-sellers': 'Best sellers',
+    'newest-arrivals': 'Newest arrivals',
+    price: 'Price',
+    'average-customer-review': 'Average customer review',
+    'number-of-comments': 'Number of comments',
+}
+
+export const categories = {
+    electronics: 'Electronics',
+    'health-and-households': 'Health & Households',
+    'home-and-garden': 'Home & Garden',
+    clothing: 'Clothing',
+    hobbies: 'Hobbies',
+    others: 'Others',
```

Comment:
Why are some categories in quotes but some are not?

Reply:
I am actually not sure about that because this file was written by @MehdiSaffar on search page pull request actually. I just copied it

Reply:
I wrote them with quotes but prettier removes quotes for single-word keys. The only difference in syntactical, the key won't actually change.